swagger.yaml

```diff
@@ -622,6 +622,17 @@ definitions:
         description: The date that the employee accepted or denied to have their salary information shared.
       retirementCode:
         type: string
+      i9FormCode:
+        type: string
+        description: Code to indicate the status of the employee's I9 form.
+      i9Date:
+        type: string
+        format: date
+        description: The date that the I9 form was received.
+      i9ExpirationDate:
+        type: string
+        format: date
+        description: The date the I9 form expires.
```

Comment:
Could you give examples of the dates? It's mainly for users to know what format of date will get by just reading the swagger/openapi specification.

Reply:
I'd rather leave that up to the client using the specification. `format: date` specifically refers to this format: 2018-12-31 https://tools.ietf.org/html/rfc3339#section-5.6

For example, the swagger editor will auto populate an example date in the example json based on the date format (`date` or `date-time`)